openfl/net/URLRequest.hx

```diff
@@ -13,7 +13,10 @@ import openfl.utils.ByteArray;
 	public var requestHeaders:Array<URLRequestHeader>;
 	public var url:String;
 	public var userAgent:String;
-	
+
+	#if (js && html5)
```

Comment:
`#if js` could be Node.js. In the past, our documentation was generated using C++ and -Dhtml5, since a true JS build did not unify. However, now our documentation does use real Flash and JS builds